### PR TITLE
Step 826 migrate xcode project

### PR DESCRIPTION
**Pull request by godrei**
Migrate from `bitrise-io/xcode-project` to `bitrise-io/go-xcode/xcodeproject`.

- Import path updates
- Go deps update

### CODE REVIEW

File: config/stack.go [hunk 1]
```diff
@@ -12,15 +12,7 @@ func Stacks() []string {
 		"osx-vs4mac-beta",
 		"osx-vs4mac-previous-stable",
 		"osx-vs4mac-stable",
-		"osx-xcode-10.1.x",
```
Comment (istvankovacs-bitrise) [inline config/stack.go:15]: Would you mind elaborating why did you remove these?

Comment (godrei) [reply config/stack.go:15]: This is a hardcoded list of available stacks, the list has changed: https://github.com/bitrise-io/bitrise.io/tree/master/system_reports

Comment (istvankovacs-bitrise) [reply config/stack.go:15]: Makes sense, thanks for the clarification!

**Review by istvankovacs-bitrise — APPROVED** [review]
